zips/zip-0317.rst

```diff
@@ -8,7 +8,11 @@
            Kris Nuttycombe
            Jack Grigg
            Francisco Gindre
-  Status: Active
+           Daniel Benarroch
+           Jonathan Rouach
+           Pablo Kogan
+           Vivek Arte
```

Comment:
If one of yous wants to add yourself as an Owner, that would be fine from my point of view. (Speaking as an existing Owner.)

Reply:
I suppose this is not relevant for now, since we aren't editing ZIP 317